app/presenters/default_deadline_date_time.rb

```diff
@@ -0,0 +1,13 @@
+# used for computing the default deadline with an offset
+class DefaultDeadlineDateTime
+  attr_reader :dc_time
+
+  def self.parse(start_time_str, day_offset)
+    time = DcTimePresenter.parse(start_time_str)
```

Comment:
I wanted to make sure that the right type of parse (ie, the one in DcTimePresenter is called vs. Time.parse), although I could just call that.